redirects.yaml

```diff
@@ -6,3 +6,4 @@
 /css/(?P<path>.*): /static/css/{path}
 /js/(?P<path>.*): /static/js/{path}
 /docs/patterns/article-pagination: /docs/patterns/pagination
+/showcase: /
```

Comment:
Great little touch!

Reply:
Shouldn't this path return deleted? As it is removed content not moved.

Reply:
@anthonydillon oh - hadn't considered that, I thought it was better to send users _somewhere_, plus I didn't know about `deleted.yaml` till just now. I'll get another PR up for that.

Reply:
Oh it's definitely better than letting it 404 but being honest is always best :)